Makefile

```diff
@@ -78,6 +79,15 @@ $(warning No code-sign identity found (CODE_SIGN_IDENTITY is not set))
 endif
 endif
 
+ifeq ($(TARGET_PLATFORM),win32)
+ifndef CODE_SIGN_CERTIFICATE
+$(warning No code-sign certificate found (CODE_SIGN_CERTIFICATE is not set))
+endif
+ifndef CODE_SIGN_CERTIFICATE_PASSWORD
+$(warning No code-sign certificate password found (CODE_SIGN_CERTIFICATE_PASSWORD is not set))
+endif
```

Comment:
As I said [here](https://github.com/resin-io/etcher/pull/968#issuecomment-266475641) I think it'd make sense for the "standard build" e.g. `make electron-installer-dmg` to produce unsigned images (and have them also be snapshot versions by default), and then have a special e.g. `signed-electron-installer-dmg` target to create (production-version?) files in the `signed/` subdirectory, and only do the check-for-certificate-vars step as part of the `signed-electron-installer-dmg` rules, and upgrade it from a warning to an error.
(If that all makes sense? :wink: )